modules/file-system/parallelstore/scripts/install-daos-client.sh

```diff
@@ -64,15 +64,17 @@ EOF
 
 	# For Ubuntu 22.04 and debian 12,
 	elif { [ "${OS_ID}" = "ubuntu" ] && [ "${OS_VERSION}" = "22.04" ]; } || { [ "${OS_ID}" = "debian" ] && [ "${OS_VERSION_MAJOR}" = "12" ]; }; then
+		# shellcheck disable=SC2034
+		DEBIAN_FRONTEND=noninteractive
 
 		# 1) Add the Parallelstore package repository
-		curl https://us-central1-apt.pkg.dev/doc/repo-signing-key.gpg | apt-key add -
-		echo "deb https://us-central1-apt.pkg.dev/projects/parallelstore-packages v2-6-deb main" | tee -a /etc/apt/sources.list.d/artifact-registry.list
+		curl -o /etc/apt/trusted.gpg.d/us-central1-apt.pkg.dev.gpg https://us-central1-apt.pkg.dev/doc/repo-signing-key.gpg
```

Comment:
I think the file extension should be `.asc` for clarity since gpg is usually reserved for the binary format. The Artifact Registry team is not following convention.

I would consider re-testing this change on Debian 12 without A3 VMs to be sufficient.

Reply:
Updated. Triggered test which runs on base debian 12.